app/mailers/order_mailer.rb

```diff
@@ -1,155 +1,124 @@
 class OrderMailer < BaseMailer
-  def buyer_confirmation(order)
-    @market = order.market
-    return if @market.is_consignment_market?
-
-    @order = BuyerOrder.new(order)
-
-    to_list = recipient_list(order)
-
-    if !to_list.blank?
-      mail(
-        to: to_list,
-        subject: "Thank you for your order"
-      )
-    end
-  end
+  def market_manager_order_updated(order)
+    return if order.market.is_consignment_market?
 
-  def seller_confirmation(order, seller)
     @market = order.market
-    return if @market.is_consignment_market?
-
-    @order = SellerOrder.new(order, seller) # Selling users organizations only see
-    @seller = seller
-
-    to_list = seller.
-                users.
-                confirmed.
-                map { |u| u.enabled_for_organization?(seller) && !u.pretty_email.nil? ? u.pretty_email : nil}.
-                compact
-
-    if !to_list.blank?
-      mail(
-        to: to_list,
-        subject: "New order on #{@market.name}"
-      )
-    end
+    @order = BuyerOrder.new(order) # Market Managers should see all items
+    mail(
+      to: order.market.managers.map(&:pretty_email),
+      subject: "#{@market.name}: Order #{order.order_number} Updated",
+      template_name: 'order_updated'
+    )
   end
 
   def market_manager_confirmation(order)
-    @market = order.market
-    return if @market.is_consignment_market?
+    return if order.market.is_consignment_market?
 
+    @market = order.market
     @order = BuyerOrder.new(order) # Market Managers should see all items
-
     mail(
       to: order.market.managers.map(&:pretty_email),
       subject: "New order on #{@market.name}"
     )
   end
 
   def invoice(order_id)
-
     @order  = BuyerOrder.new(Order.find(order_id))
-    @market = @order.market
-    return if @market.is_consignment_market?
+    return if @order.blank? || @order.market.is_consignment_market?
+    to_list = recipient_list(@order.organization)
+    return if to_list.blank?
 
-    attachments["invoice.pdf"] = {mime_type: "application/pdf", content: @order.invoice_pdf.try(:data)}
+    @market = @order.market
+    attachments['invoice.pdf'] = {mime_type: 'application/pdf', content: @order.invoice_pdf.try(:data)}
+    mail(
+      to: to_list,
+      subject: 'New Invoice'
+    )
+  end
 
-    to_list = recipient_list(@order)
-    if !to_list.blank?
+  def buyer_confirmation(order)
+    ensure_buysell_and_list(order, order.organization) do |to_list|
+      @order = BuyerOrder.new(order)
       mail(
         to: to_list,
-        subject: "New Invoice"
+        subject: 'Thank you for your order'
```

Comment:
This change is a bit misleading, I moved some of the methods around and with the simplification of the methods, the diff algorithm isn't making it clear that this is actually a different method.